### ISSUE TITLE
Using @html is a security vulnerability

### ISSUE THREAD

**Issue by olehmisar** [zerodevx/svelte-toast#88]
Rendering un-sanitised html via `@html` is a potential xss (cross site scripting) vulnerability if that html is provided by a user. Consider removing support of html toast messages.

**Comment by zerodevx** [comment]
Yes. That will be done in the [next](https://github.com/zerodevx/svelte-toast/pull/87) release.
